src/aioftp/client.py

```diff
@@ -674,6 +717,11 @@ async def login(
                 censor_after=censor_after,
             )
 
+        self.logged_in = True
+
+        if self._upgraded_to_tls:
+            await self._send_tls_protection_commands()
```

Comment:
I don't think we need to upgrade to tls implicitly at all. What this case is about?

Reply:
If I understood correctly the author's implementation, this is needed to support the authentication to happen before or after the `upgrade_to_tls`, but those protection commands can only be sent after the login itself.